### DISCUSSION TITLE
How can I build project quicker

### DISCUSSION THREAD

**Discussion by root-skaii** [vercel/community#2826] [Help] [answered]
### Summary

I am using FLASK project and using local host for testing. I've tired of installing dependencies on every edit. How can I use old cached dependencies while building project?

### Steps to Reproduce

I've tried some commands, but they are not exist:
`vercel dev --force`,
`vercel dev --max-workers 1`;

**Comment by smaeda-ks** [comment, marked as answer]
if this is about your local dev environment, you could override your "dev command" via the UI:
https://vercel.com/docs/concepts/deployments/configure-a-build#development-command

and run Flask with some debug options (run `vercel pull` first after you updated the dev command above):
- https://flask.palletsprojects.com/en/2.3.x/cli/#debug-mode
- https://sentry.io/answers/auto-reloading-a-flask-application/
